### PR TITLE
refactor(kafka_schema): simplify kafka schema import

### PR DESCRIPTION
**Pull request by ivan-savciuc**
## About this change—what it does

Simplify Kafka schema import logic 

discovered in #1245

### CODE REVIEW

**Review by Serpentiel — APPROVED** [review]
lgtm, this is also a hack, but I like it more than the other hack, especially considering that it was [disliked by the devs of Terraform](https://github.com/hashicorp/terraform-plugin-sdk/issues/1005#issuecomment-1623695760)

**Review by Serpentiel — APPROVED** [review]
lgtm

**Review by Serpentiel — APPROVED** [review]

File: internal/sdkprovider/service/kafka/kafka_schema.go [hunk 1]
```diff
@@ -254,7 +257,7 @@ func resourceKafkaSchemaRead(_ context.Context, d *schema.ResourceData, m interf
 		}
 	} else {
 		// only update if was set to not empty values by the user
-		if _, ok := d.GetOk("compatibility_level"); ok || schemautil.IsImportingResource(d) {
+		if _, ok := d.GetOk("compatibility_level"); ok {
```
Comment (byashimov) [inline internal/sdkprovider/service/kafka/kafka_schema.go:260]: Please leave a comment why do we do so ("only update if was set to not empty values by the user")

File: internal/sdkprovider/service/kafka/kafka_schema.go [hunk 2]
```diff
@@ -94,7 +94,13 @@ func ResourceKafkaSchema() *schema.Resource {
 		ReadContext:   resourceKafkaSchemaRead,
 		DeleteContext: resourceKafkaSchemaDelete,
 		Importer: &schema.ResourceImporter{
-			StateContext: schemautil.ImportStatePassthroughContext,
+			StateContext: func(ctx context.Context, d *schema.ResourceData, i interface{}) ([]*schema.ResourceData, error) {
+				if err := d.Set("compatibility_level", "importing"); err != nil {
+					return nil, fmt.Errorf("cannot set temporary value for compatibility_level: %w", err)
+				}
+
+				return schema.ImportStatePassthroughContext(ctx, d, i)
```
[the remark this answers is not in the document]
Comment (ivan-savciuc) [reply internal/sdkprovider/service/kafka/kafka_schema.go]: simplified this even further

File: internal/sdkprovider/service/kafka/kafka_schema.go (hunk 1 above)
Comment (ivan-savciuc) [reply internal/sdkprovider/service/kafka/kafka_schema.go]: please check again I simplified it further

**Comment by byashimov** [comment]
@ivan-savciuc this [test](https://github.com/aiven/terraform-provider-aiven/blob/main/internal/sdkprovider/service/kafka/kafka_schema_test.go#L20) must fail now. Have you tried to run it?

**Review by byashimov — APPROVED** [review]
lgtm

**Pull request merged**